ui/app/js/controllers/rules_controller.js

```diff
@@ -321,6 +321,39 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
     });
   };
 
+  $scope.openNewScheduledDuplicateModal = function(rule) {
+
+    var modalInstance = $modal.open({
+      templateUrl: 'rule_scheduled_change_modal.html',
+      controller: 'NewRuleScheduledChangeCtrl',
+      size: 'lg',
+      backdrop: 'static',
+      resolve: {
+        scheduled_changes: function() {
+          return [];
+        },
+        sc: function() {
+          sc = angular.copy(rule);
```

Comment:
When I tried to duplicate a Rule that only had a Scheduled Change, this ended up not prefilling any data (because `rule.product` et. al is null). It would be good to be able to duplicate pending inserts as well. Can you update this to copy `rule.scheduled_change` instead of `rule` when `rule.scheduled_change.change_type` is `insert`? Scheduled changes have additionals as well - so you'll need to be sure to strip away `sc_id`, `sc_data_version`, `when`, `complete`, and `change_type` in this case.

Reply:
Can you give me the rule id of any rule which has only has a  Scheduled Change so that I can test it??

Reply:
There usually isn't any in the sample data - you can create one with the "Schedule a new Rule" button at the top of the Rules page.